### PR TITLE
Fix issue 52

### PR DESCRIPTION
**Pull request by thenewguy**
Added ability to override gateway ip via environment variables

Fixes https://github.com/tox-dev/tox-docker/issues/52

### CODE REVIEW

**Comment by thenewguy** [comment]
I have confirmed this fixed the issue in my environment.  This is ready for merge

File: tox_docker.py [hunk 1]
```diff
@@ -47,13 +48,21 @@ def _newaction(venv, message):
...
 def _get_gateway_ip(container):
-    if sys.platform == "darwin":
-        # per https://docs.docker.com/v17.12/docker-for-mac/networking/#use-cases-and-workarounds,
-        # there is no bridge network available in Docker for Mac, and exposed ports are made
-        # available on localhost (but 0.0.0.0 works just as well)
-        return "0.0.0.0"
-    else:
-        return container.attrs["NetworkSettings"]["Gateway"] or "0.0.0.0"
+    host = os.getenv('TOX_DOCKER_GATEWAY_HOST')
+    ip = os.getenv('TOX_DOCKER_GATEWAY_IP')
```
Comment (dcrosta) [inline tox_docker.py:52]: How about 1 env var, `TOX_DOCKER_GATEWAY`? I don't think anything cares that the gateway is actually an IP address, certainly not anything in tox-docker itself. (We use this only for testing port readiness, which can work as easily with a hostname as an IP...)

Comment (thenewguy) [reply tox_docker.py:52]: Sounds good to me

**Comment by thenewguy** [comment]
https://github.com/tox-dev/tox-docker/pull/54 replaces this PR

**Pull request closed without merging**